src/util/formatTrialPrice.js

```diff
@@ -0,0 +1,11 @@
+const formatTrialPrice = (price, tax, incTax) => {
```

Comment:
- let's also rename `formatTrialPrice` to something like `calculateGrossPriceForFreeOffer`
- rename params to: netPrice, taxRate, customerPriceInclTax